lib/buffer.js

```diff
@@ -101,6 +104,13 @@ const {
   addBufferPrototypeMethods
 } = require('internal/buffer');
 
+const TypedArrayPrototype = ObjectGetPrototypeOf(Uint8Array.prototype);
```

Comment:
```suggestion
const Uint8ArrayPrototype = ObjectGetPrototypeOf(Uint8Array.prototype);
```
`Uint8Array.prototype.constructor.name` is Uint8Array, not TypedArray.

Reply:
This line is correct as-is. The prototype of `Uint8Array.prototype` is the `%TypedArray%.prototype`.

Reply:
@devsnek is correct, this is the prototype of the prototype, which is `TypedArray`.